src/watcher.js

```diff
@@ -71,16 +74,20 @@ function processEvents(events) {
   }
 }
 
+async function getLastBlockToProcess() {
+  const lastBlockNumber = await getBlockNumber(web3Instance)
+  return lastBlockNumber - requiredBlockConfirmations + 1
+}
+
 async function main({ sendToQueue }) {
   try {
-    const lastBlockNumber = await getBlockNumber(web3Instance)
-    if (lastBlockNumber === lastProcessedBlock) {
+    const lastBlockToProcess = await getLastBlockToProcess()
+    if (lastBlockToProcess === lastProcessedBlock) {
```

Comment:
Now `lastBlockToProcess` could be less than `0` if the bridge is run on a just created testnet (when `requiredBlockConfirmations` is greater than a latest block number).
Will `bridgeContract.getPastEvents(..)` handle this properly?

Reply:
I'm not sure `bridgeContract.getPastEvents(..)` will handle this properly, I'll add a fix to prevent getting the events if `lastBlockToProcess` is less than `lastProcessedBlock`